Smartsheet.Net.Standard/Entities/Share.cs

```diff
@@ -0,0 +1,21 @@
+using System;
+using Smartsheet.Net.Standard.Interfaces;
+
+namespace Smartsheet.Net.Standard.Entities
+{
+    public class Share : ISmartsheetObject
```

Comment:
You may actually need a Build() method here as well. I see that not all of these properties are included in the options you can include in the request body. The API may ignore extra parameters or it may explicitly fail. Can you test by just sending a raw call with Postman or something that includes everyone one of these properties?

Reply:
Added build method and iterations on the share methods.